### PR TITLE
Allen/qr code

### PR DESCRIPTION
**Pull request by algtan**
Added QR code component to snack bar app:

1. Installed `qrcode.react`, which is a package to create QR codes in React
2. Added QR code component to the `CodeScreen` component with styling to match design
3. Added a state variable for the QR code's value to `MainScene`. QR code will be updated anytime the user select's a new currency, or when the exchange rates change.

### CODE REVIEW

File: src/MainScene.tsx [hunk 1]
```diff
@@ -32,7 +32,8 @@ export function MainScene(): JSX.Element {
   const isInitialRender = useRef(true) // Create a mutable ref object to keep track of initial render
   const [usdToCoinRates, setUsdToCoinRates] = useState(initRates) // State variable to keep track of the exchange rates
   const [showCodeScreen, setShowCodeScreen] = useState(false) // State variable to determine whether to show the code screen
-  const [coinSelection, setCoinSelection] = useState('') // State variable that keeps track of coin selection
+  const [coinSelection, setCoinSelection] = useState('BTC') // State variable that keeps track of coin selection
```
Comment (thehobbit85) [inline src/MainScene.tsx:35]: Get the default currency from the `config` file.

File: src/components/CodeScreen.tsx [hunk 2]
```diff
@@ -25,7 +36,12 @@ export function CodeScreen(props: CodeScreenProps): JSX.Element {
     <>
       <div style={codeScreenStyle}>
         <div style={textStyle}>YOU'VE SELECTED {props.coinSelection}</div>
-        <BackButton label="BACK" handleClick={handleBackBtnClick} />
+        <div style={qrCodeBorderStyle}>
```
Comment (thehobbit85) [inline src/components/CodeScreen.tsx:39]: Separate the QR into its own component.

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
See comments

File: src/MainScene.tsx (hunk 1 above)
Comment (algtan) [reply src/MainScene.tsx]: No longer going to use a default currency. Will revert back to how it was previously with an empty string.

File: src/components/CodeScreen.tsx (hunk 2 above)
Comment (algtan) [reply src/components/CodeScreen.tsx]: Updated per comment. Created component `PaymentCode`.

**Comment by algtan** [comment]
This PR will be closed for a new one to clean up activity history.

**Comment by algtan** [comment]
This PR is superseded by PR https://github.com/EdgeApp/edge-snack-bar-app/pull/9

**Pull request closed without merging**